<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonFSend">
     <property name="text">
      <string>发送文件</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonSend">
     <property name="text">
      <string>发送文本</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="buttonClose">
     <property name="text">
      <string>关闭服务器</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="textEditRead">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditWrite"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="buttonMediaInfo">
     <property name="text">
      <string>获取视频信息</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonMedia">
     <property name="text">
      <string>查看视频</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
